src/select/select_value.rs

```diff
@@ -0,0 +1,35 @@
+use crate::select::JsonPathError;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum SelectValueType {
+    Undef,
```

Comment:
The `Undef` value is not necessary, as explained in another comment.

Reply:
Removed